src/python-bindings/collector.cpp

```diff
@@ -14,6 +14,12 @@
 
 #include "module_lock.h"
 
+// http://docs.python.org/3/c-api/apiabiversion.html#apiabiversion
+#if PY_MAJOR_VERSION >= 3
+   #define PyInt_Check(op)  PyNumber_Check(op)
+   #define PyString_Check(op)  PyBytes_Check(op)
```

Comment:
This was taken from `classad.cpp` and `classad_parsers.cpp` to match the existing code there:

https://github.com/htcondor/htcondor/blob/master/src/python-bindings/classad.cpp#L23-L26

https://github.com/htcondor/htcondor/blob/master/src/python-bindings/classad_parsers.cpp#L10-L13

I guess if you don't like it though we can define the obviously-named compat macro wrappers in `python_bindings_common.h` and replace all uses in the .cpp files...